### PR TITLE
DEV-3429 Show all object data in a pragmatic way; Doing proper reset

### PR DESCRIPTION
**Pull request by Martin01248**
I assume there is no review app
but this code changes are already live.
See here
https://akariondevelopers.github.io/open-data-integrity-verification/

Here are 2 notarization PDFs for testing

[Feuer-notarization (2).pdf](https://github.com/AkarionDevelopers/open-data-integrity-verification/files/10383469/Feuer-notarization.2.pdf)
[Korruptionsverdacht bei Vice-Direktorin Marlie Muster-notarization (1).pdf](https://github.com/AkarionDevelopers/open-data-integrity-verification/files/10383470/Korruptionsverdacht.bei.Vice-Direktorin.Marlie.Muster-notarization.1.pdf)

Before
Screenshot 2023-01-10 at 15 07 29

After
Screenshot 2023-01-10 at 15 08 24

### CODE REVIEW

File: app/index.js [hunk 1]
```diff
@@ -258,6 +258,8 @@ function updateFileList() {
 
 function resetFileList() {
   $fileList.innerHTML = '';
+  data.set('data', []);
+  data.set('isVerified', []);
```
Comment (Martin01248) [inline app/index.js:262]: I found this bug.
Reset was not properly done.
Before 2nd notarization showed data from 1st one.
Now it shows correct data.

File: app/helper.js [hunk 2]
```diff
@@ -219,12 +219,24 @@ function formatDate(date) {
 }
 
 function getObjectDataValue(objectData) {
-  if (
-    objectData == null ||
-    objectData.length == 0 ||
-    objectData.toString() == '[object Object]'
-  ) {
-    return '-';
+  if (objectData == null || objectData.length === 0) {
```
Comment (manuelmeindl) [inline app/helper.js:222]: comparisons in JS should always be done with three === so the type is also considered.

**Review by manuelmeindl — CHANGES_REQUESTED** [review]
just a small finding, rest looks fine!

File: app/helper.js (hunk 2 above)
Comment (Martin01248) [reply app/helper.js]: Fixed

**Review by manuelmeindl — APPROVED** [review]

**Pull request merged**